initialization/snyk.sh

```diff
@@ -1,19 +1,45 @@
+#!/bin/bash
+
+[ -z "$SNYK_TOKEN" ] && { echo "Set 'context: snyk' in your .circleci/config.yml workflow"; exit 1; }
+
 LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/snyk/snyk/releases/latest)
 LATEST_VERSION=$(echo $LATEST_RELEASE | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
 BINARY_URL="https://github.com/snyk/snyk/releases/download/$LATEST_VERSION/snyk-linux"
 
-# [ -z "$SNYK_LEVEL" ] && { echo "Set SNYK_LEVEL = 'RPRT' in the CircleCI environment variables settings"; exit 1; }                                                           
-[ -z "$SNYK_TOKEN" ] && { echo "Set 'context: snyk' in your .circleci/config.yml workflow"; exit 1; }
+curl -sL "$BINARY_URL" -o snyk
+[ ! -f snyk ] && { echo "Snyk failed to download!"; exit 0; }
+chmod +x snyk
 
-curl -sL "$BINARY_URL" -o snyk-linux
+severity_threshold="${severity_threshold:-low}" # by default show all vulns
+fail_on="${fail_on:-never}" # by default never fail (backwards compatibility)
+debug="${debug:-false}" # debug output is messy
 
-[ ! -f snyk-linux ] && { echo "Snyk failed to download!"; exit 0; }
+if [[ "${fail_on}" = "never" ]]; then
+  fail_on=all
+  NEVER_FAIL=true 
+fi
+
+flags=(
+  "--severity-threshold=$severity_threshold"
+  "--fail-on=$fail_on"
+  "--org=$org"
+)
+
+monitor_flags=(
+  "--org=$org"
+)
+
```

Comment:
I didn't really know what the environment stuff was doing, do I need that? 

https://github.com/segmentio/snyk-buildkite-plugin/blob/master/hooks/command#L60-L63